_data/communication.yml

```diff
@@ -254,22 +254,22 @@ websites:
       tfa: Yes
       sms: Yes
       software: Yes
-      doc: https://slack.zendesk.com/hc/en-us/articles/204509068-Enabling-two-factor-authentication
+      doc: https://get.slack.help/hc/en-us/articles/204509068
 
     - name: SocketLabs
       url: https://socketlabs.com/
       img: socketlabs.png
       sms: Yes
       tfa: Yes
       software: Yes
-      doc: https://support.socketlabs.com/kb/145
+      doc: https://help.socketlabs.com/docs/two-step-verification
 
     - name: SparkPost
       url: https://sparkpost.com/
       img: sparkpost.png
       tfa: Yes
       software: Yes
-      doc: https://support.sparkpost.com/customer/portal/articles/1948449
+      doc: https://www.sparkpost.com/docs/my-account-and-profile/enabling-two-factor-authentication/
```

Comment:
The page won't load for me.

Reply:
Works flawlessly on my end?

Reply:
![Screen Shot](https://user-images.githubusercontent.com/3535780/61235010-c498d880-a734-11e9-81af-60ae248ed9a4.png)

Reply:
I think there might be a problem with your browser not showing the content of the page? Did you check with another one?

Reply:
It was my adblockers :man_facepalming: